build.gradle.kts

```diff
@@ -278,9 +279,11 @@ tasks.register("pythonDockerBuildPreCommit") {
   dependsOn(":sdks:python:container:py36:docker")
   dependsOn(":sdks:python:container:py37:docker")
   dependsOn(":sdks:python:container:py38:docker")
+  dependsOn(":sdks:python:container:py39:docker")
 }
 
 tasks.register("pythonLintPreCommit") {
+  // TODO(BEAM-8890): Find a better way to specify lint and formatter tasks without hardcoding py version.
```

Comment:
Just to confirm, for this Jira, I see an issue related to  updating `google-cloud-java`. I don't see anything on lint and formatter.

Reply:
I think it should be BEAM-9980

Reply:
good catch